src/resources/js/components/Authentication.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="Container">
+  
+    <h2>Authentification</h2>
+
+    <div v-if="isMessage">
+      <h3>{{ message }}</h3>
+    </div>
+
+    <form @submit.prevent="login()">
+      <label for="EnterEMail">
+        <input id="EnterEMail" type="text" v-model="input.email" placeholder="Adresse e-mail" />
+      </label>
+
+      <label for="EnterPassword">
+        <input id="EnterPassword" type="password" v-model="input.password" placeholder="Mot de passe" />
+      </label>
+
+      <input type="submit" value="Se connecter"/>
+      
+
+    </form>
+    
+
+  </div>
+</template>
+
+<script>
+  import axios from 'axios'
+  import router from '../routes'
+
+  export default {
+  name : 'Authentication',
+  props:['parent'], //for future use (can't log in if already logged in, will reroute to home)
+  data () {
+    return{
+      input: {
+        email: '',
+        password: ''
+      },
+      isMessage: false,
+      message: ''
+    }
+
+
+  },
+  methods: {
+    login() {
+      if(this.input.email != "" && this.input.password != "") { //if not blank
```

Comment:
for Guideline: use '' instead of "", [source](https://github.com/airbnb/javascript#strings--quotes) and use !== instead of != [source](https://github.com/airbnb/javascript#comparison--eqeqeq)

Reply:
it has been adapted